apps/drec-api/ormconfig.ts

```diff
@@ -0,0 +1,30 @@
+import { ConnectionOptions } from 'typeorm';
+
+const getDBConnectionOptions = (): ConnectionOptions => {
+  return process.env.DATABASE_URL
+    ? {
+        type: 'postgres',
+        url: process.env.DATABASE_URL,
+        ssl: {
+          rejectUnauthorized: false,
+        },
+      }
+    : {
+        type: 'postgres',
+        host: process.env.DB_HOST ?? 'localhost',
+        port: Number(process.env.DB_PORT) ?? 5432,
+        username: process.env.DB_USERNAME ?? 'postgres',
+        password: process.env.DB_PASSWORD ?? 'postgres',
+        database: process.env.DB_DATABASE ?? 'origin',
```

Comment:
Probably want to change this to match the .env.example.

```ts
database: process.env.DB_DATABASE ?? 'origin_drec',
```

Reply:
Actually I need to revert it to origin otherwise it won`t work..the issuer will create the migrations only in origin. So we cannot use a different db.

Reply:
Are you sure? The issuer api defaults to `origin` but if you set `DB_DATABASE` to something else it should respect that.

We've changed the DB name in a couple of other projects and it worked fine, so long as you have DB_DATABASE set.

Reply:
Pretty sure. We can discuss this but I had a call yesterday with Mateusz and Patryk and it didn`t worked..
It would be good if it did.

Reply:
Is there something else or can I merge?